content/en/_index.html

```diff
@@ -34,8 +34,14 @@ <h4 class="title">
           Get started
         </button>
       </a>
-      <a href="https://github.com/notaryproject/notation">
-        <button class="button is-medium is-dark demo">GitHub</button>
+      <a href="https://killercoda.com/notaryproject/scenario/notation">
+        <button class="button is-medium is-dark demo">Try it
```

Comment:
@kokamkarsahil  Could you set this button to open a new window since it's an external link?